lib/class-wp-theme-json-resolver.php

```diff
@@ -272,6 +272,22 @@ public static function get_core_data() {
 			}
 		}
 
+		$default_duotone_i18n = array(
```

Comment:
We no longer need these for translating the strings, with what we have at `lib/experimental-default-theme.json` is fine.